roles/wisdom_deploy_model/vars/main/resources.yml

```diff
@@ -0,0 +1,8 @@
+---
+wisdom_namespace: wisdom
+serving_runtime_name: wisdom-runtime
```

Comment:
I would move these two to the generated file `roles/wisdom_deploy_model/defaults/main/config.yml` with the `@AnsibleConstant` annotation in the Python entrypoint:
```
@AnsibleConstant("<description>", "<var name>", "<default value>")
```
eg
```
    @AnsibleConstant("Namespace in which the Wisdom model will be deployed", "namespace", "wisdom")
    @AnsibleConstant("...", "serving_runtime_name", "wisdom-runtime")
```
mind that the generated name for the variables will become `wisdom_deploy_model_namespace` and `wisdom_deploy_model_serving_runtime_name`